tests/js/client/load-balancing/load-balancing-transactions-noauth-cluster.js

```diff
@@ -250,6 +254,69 @@ function TransactionsSuite () {
       }
     },
 
+    testUseTransaction: function () {
+      let trx;
+      try {
+        const obj = { collections: { read: cn } };
+        let result = sendRequest('POST', "/_api/transaction/begin", obj, true);
+        assertEqual(result.status, 201);
+        assertFalse(result.body.result.id === undefined);
+        trx = result.body.result.id;
+        
+        // use trx on different coord to run a query
+        const query = `FOR doc IN ${cn} RETURN doc`;
+        result = sendRequest('POST', "/_api/cursor", { query }, false);
```

Comment:
The query fully consumes the cursor at once. This is fine, but we should also add a test with a cursor that is queried incrementally, e.g. one result row at a time. This should be possible by giving the query a `batchSize` upon creation, and then issuing follow-up requests to `/_api/cursor/<id>`. This should work on both the correct coordinator and the incorrect one (which should do the redirect).